packages/create/src/generators/wc-lit-element/templates/partials/README.testing.md

```diff
@@ -0,0 +1,35 @@
+## Testing with Karma
+To run the suite of karma tests, run
+```bash
+<%= scriptRunCommand %> test
+```
+or to run them in compatibility mode for legacy browsers
+```bash
+<%= scriptRunCommand %> test:compatibility
+```
+
+To run the tests in watch mode (for <abbr title="test driven development">TDD</abbr>, for example), run
```

Comment:
Can you mention that you can visit http://localhost:9876/debug.html while in watch mode to debug in the browser?

Reply:
TY, added.